### DISCUSSION TITLE
Lazy loading image tiles

### DISCUSSION THREAD

**Discussion by alex-hsa** [Leaflet/Leaflet#8624] [Q&A]
All of our Leaflet maps are located on the bottom of pages and therefore below the fold for most users.
To increase page performance we would like to defer loading the map tile images until they are needed in the viewport.

I searched the Leaflet documentation but couldn't find anything about lazy-loading images. 
If this has been documented somewhere already could you please point me to the proper resource?

Thank you very much, Alex

**Comment by jonkoops** [comment]
You could place the map container on the page and use the [Intersection Observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) to defer the initialization of the map itself.
